app/backend/gwells/fixtures/wellsearch-codetables.json

```diff
@@ -1678,7 +1692,7 @@
             "update_user": "ETL_USER",
             "update_date": "2017-07-01T08:00:00Z",
             "description": "Unknown",
-            "display_order": 110,
+            "display_order": 120,
```

Comment:
I don't know if it really matters for the fixtures but we might as well leave the display_order for existing data alone and just use an unused number like 101 for the new fixture.